LabTools/DataManagement.py

```diff
@@ -215,7 +218,7 @@ def run(self):
         self.stop()
 
     def set_script(self, script_fname):
-        self.script_file_name = script_fname
+        self.script_file_name = os.path.abspath(script_fname) # FIX TO ALLOW RELATIVE PATHS
```

Comment:
good :)

Reply:
Ok good, however I don't believe I specifically stated that anywhere aside from a single commit message